backend/Board.py

```diff
@@ -1,12 +1,12 @@
 # import pygame as pg
 from backend.Cell import Cell, CellType
 
+
 class Board:
     def __init__(self, size):
         self.board_size = size
         self.create_initial_board(size)
-        
-                       
+
```

Comment:
Curious -- are these empty line changes from the auto-formatters?

Reply:
Yeah thats correct. We can adjust it but the AI configured it to match supposedly the Microsoft formatting.